From 9514618515e5bbb25c9ff6da4bb268cbc06af9f9 Mon Sep 17 00:00:00 2001
From: Matthew Fatheree <Matthew.Fatheree@belkin.com>
Date: Sat, 19 Apr 2014 00:07:04 +0700
Subject: [PATCH 04/34] Add Device tree name of Mamba board to target

Signed-off-by: Matthew Fatheree <Matthew.Fatheree@belkin.com>
---
 target/linux/mvebu/image/Makefile |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/target/linux/mvebu/image/Makefile b/target/linux/mvebu/image/Makefile
index e89530a..f8c9465 100644
--- a/target/linux/mvebu/image/Makefile
+++ b/target/linux/mvebu/image/Makefile
@@ -10,6 +10,10 @@ include $(INCLUDE_DIR)/image.mk
 TARGET_DTBS := armada-xp-db armada-370-db armada-xp-openblocks-ax3-4 armada-370-mirabox \
 		armada-370-rd armada-xp-gp
 
+ifeq ("$(PROFILE)","Mamba")
+TARGET_DTBS := armada-xp-mamba
+endif # PROFILE == Mamba
+
 LOADADDR:=0x00008000
 
 JFFS2_BLOCKSIZE = 128k
-- 
1.7.9.5

